src/lib/apis/temple/endpoints/evm/api.utils.ts

```diff
@@ -5,7 +5,9 @@ const DEFAULT_NATIVE_TOKEN_ADDRESS = '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee
 const chainIdNativeTokenAddressRecord: Record<number, string> = {
   // Polygon Mainnet
   137: '0x0000000000000000000000000000000000001010',
-  // Polygon Testnet
+  /** Polygon Testnet
+   * @deprecated https://www.alchemy.com/blog/polygon-mumbai-testnet-deprecation
+   */
   80001: '0x0000000000000000000000000000000000001010'
```

Comment:
Should we add [Amoy testnet](https://chainlist.org/?search=amoy&testnets=true) in this case?